mitosheet/src/mito/types.tsx

```diff
@@ -805,6 +808,9 @@ export interface MitoConfig {
     [MitoEnterpriseConfigKey.DISPLAY_SCHEDULING]: boolean,
     [MitoEnterpriseConfigKey.CUSTOM_SHEET_FUNCTIONS_PATH]: string,
     [MitoEnterpriseConfigKey.CUSTOM_IMPORTERS_PATH]: string,
+    [MitoEnterpriseConfigKey.JUPYTER_LOG_SERVER_URL]: string,
+    [MitoEnterpriseConfigKey.JUPYTER_LOG_SERVER_BATCH_INTERVAL]: string,
```

Comment:
We decided that we were going to expose this so that we didn't need to configure for them. I'd rather expose this to remove back and forths with enterprises trying to set this. They could iterate themselves in 5 minutes to find the right one or we could go back and forth for weeks.